docs/troubleshoot/security.md

```diff
@@ -2,6 +2,78 @@
 
 Refer to vulerability management and release documentation [here](https://github.com/ratify-project/ratify/blob/dev/SECURITY.md).
 
+## Signature Validation
+
+Ratify signs all dev images and dev helm OCI artifacts with Notary Project and Sigstore Cosign signatures.
```

Comment:
Why do we only mention signature validation for dev assets? Will the official release assets be verifiable? If so, should we state that the plan to make the official release assets verifiable?

Reply:
Sure I can add that a note saying that release assets will be supported in near future